src/components/screens/DocsScreen/CodeSnippets.tsx

```diff
@@ -61,18 +69,56 @@ export function PureCodeSnippets(props) {
   return <StyledDesignSystemCodeSnippets {...props} />;
 }
 
-export function CodeSnippets({ paths, ...rest }) {
+const MissingMessaging = styled.div`
+  background-color: #fdf5d3;
+  padding: 10px 16px;
+  border-bottom: 1px solid ${color.border};
+  font-size: ${typography.size.s2}px;
+  line-height: 20px;
+`;
+
+export function ModuleComponentWithMessage({
+  currentFramework,
+  ModuleComponent,
+  withMissingMessaging,
+}) {
+  return (
+    <>
+      {withMissingMessaging && (
+        <MissingMessaging>
+          This snippet doesnt exist for {stylizeFramework(currentFramework)} yet.{' '}
+          <Link
+            appearance="secondary"
+            href="https://github.com/storybookjs/storybook/tree/next/docs"
```

Comment:
Is it possible to get a link to the actual file here?

Also, I wonder if this should be `master` given that is the branch that is deployed?

Reply:
Any thoughts on that @shilman? (i.e. do we want docs PRs to `next` or `master` for snippet contributions?)

Reply:
The file doesn't exist yet, hence why we can't link to it. I asked about this in slack & whether or not we should just link to the react snippet, but it sounds like there is a plan to add a contributing guide for the snippets eventually. This seemed to be the interim solution. Let's see what @shilman has to say on the branch...

Reply:
Ideally it should link to `next` and we'll patch it through to `master`.

We could also link to `master` which would be more intuitive for users, but I prefer forcing everything into `next` first for consistency with our code workflow.

We need to patch changes through one way or the other regardless of where we link, so neither direction saves work for us as maintainers AFAIK.

Reply:
Ok fine by me